nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, model, config_list, optimizer=None):
                     types of nn.module you want to apply quantization, eg. 'Conv2d'
         """
         super().__init__(model, config_list, optimizer)
-        self.quant_grad = QATGrad
+        self.quant_grad = QATGrad.apply
```

Comment:
Why we have to move `apply` here instead of using it directly?

Reply:
So it is for avoiding STE in LSQ quantizer.

Reply:
Yes, it is aimed at unifying the framework of quantizers with customized gradient and quantizers with auto-grad gradient. Also, use`.apply` is the way recommended by PyTorch (see [here](https://pytorch.org/tutorials/beginner/examples_autograd/two_layer_net_custom_function.html))